node/src/lib/deploy-utils.js

```diff
@@ -126,7 +127,8 @@ class DeployUtils {
   }
 
   async pollDeploymentStatus(deployment, shouldProcessDeploymentSteps = true) {
-    const MAX_TIME_IN_SECONDS = 10800; // 3 hours
+    const maxTimeInSeconds = 10800; // 3 hours
+    const pollStepLogsInterval = 30000; // 30 seconds
```

Comment:
I meant, all the way up - again, so we'd be able to better track this configuration going forward.  
```typescript
const configuration = {
  pollDeploymentStatus: {
    timeout: 10800,
    interval: 30000
  }
}
```